app/client/src/components/shared/DataContent.js

```diff
@@ -113,515 +125,101 @@ function DataContent() {
     };
   }, []);
 
+  const { data, status } = useDataSourcesContext();
+
+  useLayoutEffect(() => {
+    if (status !== 'success') return;
+    const spans = Array.from(document.querySelectorAll('span'));
+    spans.forEach((span) => {
+      if (
+        !span.dataset.hasOwnProperty('glossaryTerm') ||
+        !span.dataset.hasOwnProperty('term')
+      )
+        return;
+
+      const node = document.createElement('span');
+      render(
+        <GlossaryTerm term={span.dataset.term}>{span.innerText}</GlossaryTerm>,
+        node,
+      );
+      span.parentNode.replaceChild(node, span);
+    });
+  }, [status]);
+
+  if (status === 'fetching') return <LoadingSpinner />;
+
+  if (status === 'failure') {
+    return (
+      <div css={modifiedErrorBoxStyles}>
+        <p>{dataContentError}</p>
+      </div>
+    );
+  }
```

Comment:
Can you update this to have padding around the errorbox? See the `pageErrorBoxStyles` in the `pages\WaterbodyReport.js` component.

![image](https://user-images.githubusercontent.com/46329268/215827061-b83b8bec-f2d4-462d-881b-e062e81b0d3d.png)